functions/go/set-namespace/README.md

```diff
@@ -51,6 +61,12 @@ subjects:
   - kind: ServiceAccount
     name: default # <======== using name default here
     namespace: original-namespace
+  - kind: ServiceAccount
+    name: service-account # <======== name matches above ServiceAccount
+    namespace: original-namespace
+  - kind: ServiceAccount
+    name: other-service-account # <======== will NOT be updated
+    namespace: original-namespace
```

Comment:
nit:
```yaml
subjects:
  - kind: ServiceAccount
    name: default # <=============== name default is used
    namespace: original-ns # <====== this will be updated
  - kind: ServiceAccount
    name: service-account # <======= name matches above ServiceAccount
    namespace: original-ns # <====== this will be updated
  - kind: ServiceAccount
    name: other-service-account
    namespace: original-ns # <====== this will NOT be updated
```

Reply:
Good suggestion, done!